tests/framework/pvc.go

```diff
@@ -284,24 +283,15 @@ func (f *Framework) VerifyBlankDisk(namespace *k8sv1.Namespace, pvc *k8sv1.Persi
 }
 
 // VerifySparse checks a disk image being sparse after creation/resize.
-func (f *Framework) VerifySparse(namespace *k8sv1.Namespace, pvc *k8sv1.PersistentVolumeClaim, imagePath string) (bool, error) {
+func (f *Framework) VerifySparse(namespace *k8sv1.Namespace, pvc *k8sv1.PersistentVolumeClaim, imagePath string, originalVirtualSize int64) (bool, error) {
 	var info image.ImgInfo
-	var imageContentSize int64
 	err := f.GetImageInfo(namespace, pvc, imagePath, &info)
 	if err != nil {
 		return false, err
 	}
-	// qemu-img info gives us ActualSize but that is size on disk
-	// which isn't important to us in this comparison; we compare content size
-	err = f.GetImageContentSize(namespace, pvc, imagePath, &imageContentSize)
-	if err != nil {
-		return false, err
-	}
-	if info.ActualSize-imageContentSize >= units.MiB {
-		return false, fmt.Errorf("Diff between content size %d and size on disk %d is significant, something's not right", imageContentSize, info.ActualSize)
-	}
-	fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: VerifySparse comparison: Virtual: %d vs Content: %d\n", info.VirtualSize, imageContentSize)
-	return info.VirtualSize >= imageContentSize, nil
+	fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: VerifySparse comparison: OriginalVirtual: %d vs SizeOnDisk: %d\n", originalVirtualSize, info.ActualSize)
+	// The size on disk of a sparse image is significantly lower than the image's virtual size
+	return originalVirtualSize-info.ActualSize >= units.MB, nil
```

Comment:
What is the 1MB fudge factor all about?

Reply:
Its probably to ensure we align the size on 1Mib. qemu gets unhappy if the image is not aligned on the actual storage block size.

Reply:
I think this check is a little too strict, wondering why current virtual size is not appropriate

Reply:
So I looked up the 1MiB->1MB switch.
It's because `qemu-img convert` only manages to shave off one mega**byte** for our test image
```INFO: VerifySparse comparison: OriginalVirtual: 18874368 vs SizeOnDisk: 17829888```

Reply:
So this check is specific to a particular image?

Reply:
No, you pass the virtual size of the used image as a parameter